### DISCUSSION TITLE
what does "Reset per app audio" do?

### DISCUSSION THREAD

**Discussion by GuitarBilly** [Belphemur/SoundSwitch#1461] [Q&A] [answered]
hello!
I see this item in the main tray icon menu called "Reset per app audio". What does it do?

I could not find any information about it and I would like to translate this string. I dare not click on it yet as I have SoundSwitch set up perfectly.

**Comment by Belphemur** [comment, marked as answer]
Hello,

Since Win 10 you can have application being assigned a playback and recording device.
SoundSwitch does use that feature for forcing foreground application to change audio device.

Sometimes, there are issue with those devices assignation and resetting it is needed, hence this feature.

We're going to have a better explanation in the app soon.
